roles/sensu-check/defaults/main.yml

```diff
@@ -39,6 +39,15 @@ sensu_checks:
       standalone: true
       command: "check-disk.rb -w 80 -c 90"
       service_owner: openstack
+    check_raid:
+      handler: default
+      interval: 300
+      occurrences: 1
+      standalone: true
+      command: "check-raid.sh -z critical"
```

Comment:
It is, only because I haven't found a solid way to handle the criticality aspect of with due to variable referencing complexity. This seems like an easy solution, matching how they are currently configured in defaults _which ends up ignoring any overrides_ and most likely solving the problem with tags instead.